test/extended/operators/olm.go

```diff
@@ -37,12 +36,72 @@ var _ = g.Describe("[Feature:Platform] OLM should", func() {
 			e2e.Logf("Successfully list %s", olmResources[i])
 		})
 	}
+
+	// OCP-22070
+
+	g.It("[ocp-22070]support grpc sourcetype", func() {
+		catalogsourceYF := "https://raw.githubusercontent.com/jianzhangbjz/v3-testfiles/v4.1/olm/catalogsource/image-catalogsource.yaml"
```

Comment:
Yes, thanks! For me, I don't like to use this kind of way to implement it. But, I could not find a better solution. What do you suggest? 
I will change this file repo to official one once we decide to use this solution.

Reply:
@bparees @njhale Is it available if we add these YAML files under a newly created folder in this Origin repo?

Reply:
@jianzhangbjz yes, we use bindata for this in our other tests.  e.g.:

https://github.com/openshift/origin/tree/master/test/testdata
https://github.com/openshift/origin/blob/master/test/extended/builds/build_pruning.go#L27

you'll have to run hack/update-generated-bindata.sh if you introduce new fixture content.

Reply:
(as I recall our test framework extracts the files from the bindata.go and makes them available on the filesystem when the tests execute)

Reply:
@bparees Sounds good! Thanks! I will have a try!

Reply:
@bparees How can we set up these YAML files to receive parameter when running test?

Reply:
make them openshift templates and then instantiate them?

Reply:
Yes, do you have any suggestions?

Reply:
i'm confused about the question.  You should wrap the catalogsource object in a template and parameterize it, and then use `oc process` from within your test to instantiate it with whatever parameters you need.

Reply:
Yes, thanks! One more thing. I guess it's inappropriate to submit this PR to `master` branch since it's always changing. I should submit it to `release-4.1` branch since it only covers 4.1 scenarios, right?

Reply:
i haven't looked at exactly what this test does that would make it 4.1 specific, but it's unusual to have branch-specific test cases.  Why is it not applicable to 4.2/master?